### PR TITLE
remove unnecessary cancellation token and use one in OperationContext.

### PR DESCRIPTION
**Pull request by heejaechang**
the cancellation token will be removed from ICodeCleanUpFixer in next API update

### CODE REVIEW

**Comment by heejaechang** [comment]
tagging @sharwell 

@olegtk do you know github alias for chris?

File: src/VisualStudio/Core/Def/Implementation/CodeCleanup/CodeCleanUpFixer.cs [hunk 1]
```diff
@@ -13,6 +14,13 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.CodeCleanup
     /// </summary>
     internal abstract class CodeCleanUpFixer : ICodeCleanUpFixer
     {
-        public abstract Task<bool> FixAsync(ICodeCleanUpScope scope, ICodeCleanUpExecutionContext context, CancellationToken cancellationToken);
+        [Obsolete("use one without cancellationtoken", error: true)]
+        public virtual Task<bool> FixAsync(ICodeCleanUpScope scope, ICodeCleanUpExecutionContext context, CancellationToken _)
+        {
+            // cancellation token will be removed in next API update
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/Implementation/CodeCleanup/CodeCleanUpFixer.cs]: Tracking bug or something to link to on the editor side?

Comment (heejaechang) [reply src/VisualStudio/Core/Def/Implementation/CodeCleanup/CodeCleanUpFixer.cs]: sure. I can create one.

Comment (heejaechang) [reply src/VisualStudio/Core/Def/Implementation/CodeCleanup/CodeCleanUpFixer.cs]: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/841276

**Pull request merged**